### ISSUE TITLE
2.10.0-alpha2 Release Plan

### ISSUE THREAD

**Issue by trackleft** [az-digital/az_quickstart#3346]
See https://github.com/orgs/az-digital/projects/87
See https://github.com/az-digital/az_quickstart/milestone/20

- #3331
- #3336
- #3345
- #3341
- #3348
- #3350
- #3357
- #3358
- #3340
- #2041
- #3212
- #3359

**Comment by joeparsons** [comment]
## Pick list
```
git cherry-pick 63bc31c913016cd59843d28eccd0670047ac881c
git cherry-pick 8698a9e9460fdc9f8fcc08fb1c58ce08f562f81a
git cherry-pick f0bf9bedd1639357ececc86d75b53d676de3c084
git cherry-pick eafe9d3331152155af1cf9e8a27ba75999fb424e
git cherry-pick 35ac7c422294550b20940316b568674cd403fb24
git cherry-pick b113fb967da1ba61ddfc95b136ee5fc1a764b302
git cherry-pick 28116f0e5b55ced0178c6f81fa06acf9eac9d36a
git cherry-pick ffcc8034ddd73c88902ddb73bbbc74fe4450d651
git cherry-pick 79bba6c1dfdd92234f14a022cd2bedb0e244ea46
git cherry-pick 268237e42cd1bf4d04fc41c06a76dca38f28ba2e
git cherry-pick 88f21f2b4e89cc1622871bf9b4428b4992a53f54
git cherry-pick ced706b4375cbb2c14202f5b069ac6b93e4c1396
```

## Release notes snippet

`git log 2.10.0-alpha1..HEAD --pretty='format:- %s'`
```


- Update citation-style-language/locales requirement from 0.0.15 to 0.0.19 (#3359)
- Update citation-style-language/locales requirement from 0.0.11 to 0.0.15 (#3212)
- Update citation-style-language/styles requirement from 0.0.343 to 0.0.409 (#3340)
- Closes #2037 - Make card titles configurable (#2041)
- #3266: Update drupal/config_ignore requirement from 3.2.0 to 3.3.0 (#3358)
- Update drupal/entity_embed requirement from 1.5.0 to 1.6.0 (#3341)
- Update drupal/slick requirement from 2.10.0 to 2.11.0 (#3350)
- Update drupal/token requirement from 1.13.0 to 1.14.0 (#3357)
- Fixes #3347, Fixes #3338 Allow QuickstartConfigProvider to ignore some Diff types. (#3348)
- Closes #3334 Remove modules deprecated as of 2.8.x. (#3336)
- Closes #3344 Some meta tags are missing in 2.10.x (#3345)
- Close #3330 Ensure Recurring Import Rule entity type is installed during update. (#3331)
- Back to dev.
```

**Comment by joeparsons** [comment]
https://github.com/az-digital/az_quickstart/releases/tag/2.10.0-alpha2
